ucp/_libs/core.pyx

```diff
@@ -515,11 +515,12 @@ cdef class ApplicationContext:
             event_loop.create_task(_non_blocking_mode(weakref.ref(self)))
         )
 
-    def continually_ucx_prograss(self):
+    def continually_ucx_prograss(self, event_loop=None):
```

Comment:
Is it supposed to be progress maybe? I think a better name would be `continuous_ucx_progress`.

Reply:
Right :)